src/database/schema/ChatSchema.ts

```diff
@@ -297,7 +309,7 @@ class ChatSchema implements BaseDbSchema {
       updatedAt: new Date().toISOString(),
       id: newRandomId,
       type,
-      rawJson: null,
+      rawJson,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `rawJson` field is now being set with the value of `rawJson` variable instead of `null`. This change seems fine as it allows storing additional data in the chat schema.
```diff
-      rawJson: null,
+      rawJson,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->